Community Toolbox/scripts/tests_ArrayMaxTests/tests_ArrayMaxTests.gml

```diff
@@ -19,19 +24,60 @@ function ArrayMaxTests(_run, _method) : VerrificMethodTest(_run, _method) constr
         then_result().should_be(10);
     }
     
+    static should_handle_large_array = function() {
+        given_array(array_with_100k_items);
+        when_array_max_runs();
+        then_result().should_be(99_999);
+    }
+    
+    static should_handle_positive_offset_and_length = function() {
```

Comment:
For max and min specifically, I might recommend adding some tests with a list that *isn't* monotonic. As of right now, these tests would pass if the function simply returned the last element (respectively, first element) in the range for the max (respectively, min) function.

Reply:
I changed up the arrays involved, so that min/max results positioning in subsections is more varied.